search-service/src/test/kotlin/com/egm/stellio/search/entity/web/EntityOperationHandlerTests.kt

```diff
@@ -269,57 +225,14 @@ class EntityOperationHandlerTests {
         }
     }
 
-    @Test
```

Comment:
@thomasBousselin you removed some tests but they could / should still be tested in the handler (not enough rights, non existing entity in the batch)

Reply:
We should however check those point in the entityServiceTest; If they do not exist yet.

Reply:
yes, but in the handler, it is important to check that it correctly accumulates the errors and successes for each entity that is in the batch.

Reply:
After double checking, i think (line 137) the test "update batch entity should return a 207 if JSON-LD payload contains update errors" already check this case.